sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -508,125 +527,115 @@ def _from_generated(cls, generated):
-    :ivar bool blob_tier_inferred:
-        Indicates whether the access tier was inferred by the service.
-        If false, it indicates that the tier was set explicitly.
-    :ivar bool deleted:
-        Whether this blob was deleted.
-    :ivar ~datetime.datetime deleted_time:
-        A datetime object representing the time at which the blob was deleted.
-    :ivar int remaining_retention_days:
-        The number of days that the blob will be retained before being permanently deleted by the service.
-    :ivar ~datetime.datetime creation_time:
-        Indicates when the blob was created, in UTC.
-    :ivar str archive_status:
-        Archive status of blob.
-    :ivar str encryption_key_sha256:
-        The SHA-256 hash of the provided encryption key.
-    :ivar str encryption_scope:
-        A predefined encryption scope used to encrypt the data on the service. An encryption
+        with flexible latency requirements."""
+    rehydrate_priority: str
+    """Indicates the priority with which to rehydrate an archived blob"""
+    blob_tier_change_time: "datetime"
+    """Indicates when the access tier was last changed."""
+    blob_tier_inferred: bool
+    """Indicates whether the access tier was inferred by the service.
+        If false, it indicates that the tier was set explicitly."""
+    deleted: bool
+    """Whether this blob was deleted."""
+    deleted_time: "datetime"
+    """A datetime object representing the time at which the blob was deleted."""
+    remaining_retention_days: int
+    """The number of days that the blob will be retained before being permanently deleted by the service."""
+    creation_time: "datetime"
+    """Indicates when the blob was created, in UTC."""
+    archive_status: str
+    """Archive status of blob."""
+    encryption_key_sha256: str
+    """The SHA-256 hash of the provided encryption key."""
+    encryption_scope: str
+    """A predefined encryption scope used to encrypt the data on the service. An encryption
         scope can be created using the Management API and referenced here by name. If a default
         encryption scope has been defined at the container, this value will override it if the
-        container-level scope is configured to allow overrides. Otherwise an error will be raised.
-    :ivar bool request_server_encrypted:
-        Whether this blob is encrypted.
-    :ivar list(~azure.storage.blob.ObjectReplicationPolicy) object_replication_source_properties:
-        Only present for blobs that have policy ids and rule ids applied to them.
+        container-level scope is configured to allow overrides. Otherwise an error will be raised."""
+    request_server_encrypted: bool
+    """Whether this blob is encrypted."""
+    object_replication_source_properties: List["ObjectReplicationPolicy"]
+    """Only present for blobs that have policy ids and rule ids applied to them.
 
-        .. versionadded:: 12.4.0
+        .. versionadded:: 12.4.0"""
```

Comment:
Do we still want these here? I left them but not 100% sure what we want to do with these.